dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/lb/HttpEndpointBasedTokenMapSupplier.java

```diff
@@ -40,109 +39,160 @@
 
 public class HttpEndpointBasedTokenMapSupplier extends AbstractTokenMapSupplier {
 
-	private static final Logger Logger = LoggerFactory.getLogger(HttpEndpointBasedTokenMapSupplier.class);
-	
-	private static final String DefaultServerUrl = "http://{hostname}:8080/REST/v1/admin/cluster_describe";
-	private final String serverUrl;
-	private static final Integer NumRetries = 2;
+    private static final Logger Logger = LoggerFactory.getLogger(HttpEndpointBasedTokenMapSupplier.class);
 
-	public HttpEndpointBasedTokenMapSupplier(int port) {
-		this(DefaultServerUrl, port);
+    private static final String DefaultServerUrl = "http://{hostname}:8080/REST/v1/admin/cluster_describe";
+    private final String serverUrl;
+    private static final Integer NumRetriesSameNode = 2;
```

Comment:
This was there before, so I followed the conventions however, according the [Java Language Specifications](http://docs.oracle.com/javase/specs/#9367) _The names of constants in interface types should be, and final variables of class types may conventionally be, a sequence of one or more words, acronyms, or abbreviations, all uppercase, with components separated by underscore "_" characters_. SO updated.